models/labels/trader_frequencies/ethereum/trader_frequencies_ethereum.sql

```diff
@@ -0,0 +1,40 @@
+{{
+    config(
+        alias='trader_frequencies_ethereum',
+    )
+}}
+
+with
+ trader_frequencies as (
+    select
+        taker as address,
+        count(tx_hash) / datediff(max(block_time), min(block_time)) as trades_per_day
```

Comment:
Can we use block date on these? I remember that these columns were required for partitioning tables and it seems like a reasonable place to use it

Reply:
Wow! There's an enormous performance increase in my query. Thanks, Ben!